src/commands/app-configs/create.ts

```diff
@@ -83,9 +83,7 @@ should be unique.`
   }
 
   captureOptions() {
-    debug('captureOptions()')
-    const {flags} = this.parse(AppConfigsCreate)
-    this.options = flags
+    super.captureOptions()
     debug('flags: %O', flags)
```

Comment:
This is debugging information which is replicated, at least partially, in `MixCommand`. Unless it's needed for a more delicate operation, this method can be deleted.

Reply:
I removed the extra debug logs. Thanks!